ext/redhat/pxp-agent.init

```diff
@@ -46,15 +46,10 @@ start() {
 
 stop() {
     echo -n $"Stopping PXP agent: "
-    if [ "$USEINITFUNCTIONS" = "true" ]; then
-        killproc $pidopts $exec
-        RETVAL=$?
-    else
-        if [ -n "${pid}" ]; then
-            kill -TERM $pid >/dev/null 2>&1
-            RETVAL=$?
-        fi
-    fi
+    trap '' TERM
+    killall $prog 2>/dev/null
+    trap TERM
+    RETVAL=$?
```

Comment:
I'm not sure this is capturing the right error result. It might be grabbing the result of `trap`.